Cargo.toml

```
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
 members = ["crates/*"]
```

nit:
```suggestion
members = ["crates/*"]
resolver = "2"
```

Rationale is the `members` field is more important in the context of the workspace and should come first.